sklearn/preprocessing/_discretization.py

```diff
@@ -166,6 +189,34 @@ def fit(self, X, y=None):
                 " instead."
             )
 
+        # Check if subsample is needed
+        n_samples, n_features = X.shape
+
+        if self.strategy == "quantile" and self.subsample == "warn":
+            warnings.warn(
+                "In version 1.2, onwards subsample = 2e5"
+                "will be used by default. Pass subsample=None to"
+                "silence this warning for now.",
```

Comment:
Could you make sure to take the 79 available characters here.

Reply:
There also missing space at the end of the first 2 lines to have a proper string